### ISSUE TITLE
Please add default/linux/amd64/13.0/desktop/plasma/no-multilib profile

### ISSUE THREAD

**Issue by octoploid** [gentoo/kde#395]
Currently the default/linux/amd64/13.0/desktop/plasma profile
implicitly assumes muiltlib support.

Please add:
default/linux/amd64/13.0/desktop/plasma/no-multilib
for users that don't need muiltlib.

Thanks.

**Comment by kensington** [comment]
@mgorny not really related to the overlay...but as a multilib person, and thoughts about this?

**Comment by mgorny** [comment]
If you want to maintain it, sure. If you don't, I wouldn't blame you for rejecting the request. These days, the only thing really forced by the multilib profiles is multilib toolchain.

That said, we really need better profile structure.

**Comment by octoploid** [comment]
I noticed it the first time when I tried to build sys-apps/sandbox. 
This will force multilib USE flag with plasma.profile. And of course
it will fail to build on a non multilib system.

Maybe allowing the user to select multiple entries in the "eselect profile list"
and then logically merging them would help.

In the meantime adding `-multilib` to profiles/targets/desktop/plasma/use.force
seems to be sufficient for now.
